src/targets/gpu/target.cpp

```diff
@@ -131,6 +131,7 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         simplify_qdq{},
         enable_pass(not mlir_enabled(), rewrite_quantization{}),
         dead_code_elimination{},
+        eliminate_data_type{{migraphx::shape::uint8_type}, shape::float_type},
```

Comment:
You are not specifiying "ops" for which it should eliminate uint8 type. Therefore by default it would eliminate uint8 for all the ops. 

If that is what you intend to do then, you can just add `uint8` in `unsupported_type` and just do one call to `eliminate_data_type{}` instead of three that are happening here.

Reply:
That's the intent. Okay will do

Reply:
Done